src/tasks/pagination/TasksPage.tsx

```diff
@@ -177,6 +200,11 @@ class TasksPage extends PureComponent<Props, State> {
     sortDirection: Sort,
     sortType: SortTypes
   ) => {
+    const url = new URL(location.href)
+    url.searchParams.set('sortKey', sortKey)
+    url.searchParams.set('sortDirection', sortDirection)
+    history.replaceState(null, '', url.toString())
```

Comment:
First I thought passing `null` was going to overwrite the browser history, making the back button useless. But then I read some more on the docs of `replaceState` and this makes sense. 

So we want to be able to go back to a previous page in the pagination, through browser history, is that a correct interpretation?

Reply:
>So we want to be able to go back to a previous page in the pagination, through browser history, is that a correct interpretation?

@ChitlangeSahas 

Great question, love where your head's at 😄 What this lets us do is change the url without changing history. So as you paginate through results, it'll update the url with `?page=$x` but not put those in the history stack, so you don't have to hit the back button to go through your pagination history.

If you click Boards in the nav, then click Tasks, then paginate through 5 pages, then hit back, you'll be back at the dashboards page.

Reply:
Ahhh! Total opposite of what I thought it was doing, good to know! Thank you for explaining!